src/components/Filters/Filters.tsx

```diff
@@ -5,25 +5,36 @@ import { FlexContainer } from '../FlexContainer';
 import { FilterRow } from './FilterRow';
 import { FilterProps } from './Filters.types';
 import { OptionPropType } from './Select/Select.types';
+import { InputTypes } from './Filters.enums';
 
-const Filters: React.FC<FilterProps> = ({ options }) => (
+const Filters: React.FC<FilterProps> = ({ dataOptions, rows }) => (
   <FlexContainer flexDirection="column">
-    {options.map(({ id, ...rest }, index) => (
-      <FilterRow key={id} id={id} isFirstRow={index === 0} {...rest} />
+    {rows.map(({ id, ...rest }, index) => (
+      <FilterRow
+        key={id}
+        dataOptions={dataOptions}
+        id={id}
+        isFirstRow={index === 0}
+        {...rest}
+      />
     ))}
   </FlexContainer>
 );
 
 export default Filters;
 
 Filters.propTypes = {
-  options: PropTypes.arrayOf(
+  dataOptions: PropTypes.arrayOf(OptionPropType).isRequired,
+  rows: PropTypes.arrayOf(
```

Comment:
I'm missing the connection between table columns and conditions + inputTypes. 
e.g. I will select `col x` and how do the filters know that it should show me `string` input and not `enum`?

Reply:
Hm, good catch, it is not there. I will probably move input type to data options to get this mapping

Reply:
There has to dependency Data point -> Condition -> Input type

Reply:
@PrincAm I know I'm iterating this over and over but what about:
```js
interface FiltersProps {
  rows: {
    [key: dataPoint]: { // <-- column identifier
      conditions: [], // <-- list of conditions
      defaultCondition: '',
      inputType: '' // <-- this should probably accept React components also
    }
  }
}
```

Reply:
```const options = {
  dataPoint: {
    // issues count
    conditions: [
      {
        id: string,
        input:
          InputValueTypes |
          React.Element |
          {
            type: InputValueTypes | React.Element,
            props: { options },
          },
        isDefault: bool, // true
        label: string, // is at least
        value: string, // value
        operation: string,
      },
    ],
  },
};

const defaultRows = {
  operator: 'and', // 'and' | 'or'
  filters: {
    value: '',
    dataPoint: '',
    condition: '',
  },
};
// is, is not, contains, does not contain
const issuesCount = {
  conditions: [
    {
      label: 'greater than',
      value: 'gt',
      operation: 'gt'
      input: {
        type: CountFilter,
        props: {
          min: 10,
          max: 100,
        }
      },
    },
  ],
};

Reply:
@ajkl2533 One more round 🙂 API was updated as we agreed. Logic of handling between selectors and input types within `FiltersRow` will be in next PR.